public/css/style.css

```diff
@@ -67,7 +67,8 @@ small, .font_small {
     font-size: .8em;
     padding: .5em;
     text-decoration: none;
-    border: 1px solid #fff;
+    background-color: #4BA398;
+    border: 1px solid #4BA398;
```

Comment:
ok i can revert this, find something as u prefere but let's make it more "visible"